### ISSUE TITLE
GoWatchIt: Incorrectly shows "Available for streaming" on Netflix

### ISSUE THREAD

**Issue by moollaza** [duckduckgo/zeroclickinfo-spice#1813]
https://duckduckgo.com/?q=Watch+Full+metal+jacket&ia=watch

Seems like it's only available for Netflix mail? Not sure why this shows streaming is available.

/cc @jagtalon

**Comment by MrChrisW** [comment]
Easy fix in JS is: 

```js
if(item.provider_name === "Netflix" && item.category !== "online") {
    return null;
}
```

Do we want to remove these results completly or just change the way they're presented? Example mockup:
![selection_389](https://cloud.githubusercontent.com/assets/5282264/7363410/9ae5e474-edac-11e4-8001-a3eb64e610cc.png)

**Comment by moollaza** [comment]
@abeyang @chrismorast thoughts on this? Jordan noticed we say "Available for streaming" when in fact it's only available for rent via mail -- is @MrChrisW proposed wording^ OK?

**Comment by abeyang** [comment]
@moollaza @MrChrisW can we say "Available for rent", and push "rent" to the second line?

**Comment by MrChrisW** [comment]
@abeyang Happy with something like this?
![selection_391](https://cloud.githubusercontent.com/assets/5282264/7388948/2630ae0c-ee9c-11e4-9020-cb8d65188af5.png)

**Comment by abeyang** [comment]
@MrChrisW yep, please capitalize 'R'. Thanks!

**Comment by MrChrisW** [comment]
@abeyang :+1: Done #1816 Thanks!

**Comment by abeyang** [comment]
:+1: /cc @moollaza

**Comment by jagtalon** [comment]
Fixed by @MrChrisW  in #1816
